.github/workflows/publish.yml

```
@@ -266,16 +266,32 @@ jobs:
           name: vector-${{ env.VECTOR_VERSION }}-arm-unknown-linux-musleabi
           path: target/artifacts/vector*
 
-  build-x86_64-apple-darwin-packages:
-    name: Build Vector for x86_64-apple-darwin (.tar.gz)
-    runs-on: macos-latest-large
+  build-apple-darwin-packages:
+    name: Build Vector for Apple Darwin ${{ matrix.architecture }} (.tar.gz)
```

```suggestion
    name: Build Vector for ${{ matrix.architecture }}-apple-darwin (.tar.gz)
```

Realized this could be an opportunity to make the name consistent with the other jobs here too 😅